integration-tests/tests/src/tests/sync/sync-session.ts

```diff
@@ -278,7 +278,7 @@ describe("SessionTest", () => {
           const realm = new Realm(config);
           const session = realm.syncSession;
           //@ts-expect-error using internal method.
-          session._simulateError(123, "simulated error", "realm::sync::ProtocolError", false);
+          session._simulateError(123, "simulated error", "realm::sync::ProtocolError", false); // this will lead to error code 1032
```

Comment:
Should the 1st argument here also be changed to `1032` as on line 267?

Reply:
Good point. 123 isn't used by core